packages/harpguru-core/src/utils/reduce-for-new-column-bounds/determine-zoom-id.ts

```diff
@@ -0,0 +1,14 @@
+import { ZoomIds } from '../../types'
+
+export const determineZoomId = (
```

Comment:
The presence of this means that at least a component of the `useDeterminZoomId` proposed in a previous comment is going to have to be a util and not a hook so that it can be used where this is currently used.

In other words, this is probably going to have to be moved somewhere more shared and used by the `useDetermineZoomId` hook.

Reply:
The `useDetermineZoomId` was unnecessary in the end. It's enough to just use the util.